tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -187,28 +209,38 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 					return false
 				}
 
-				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
+				return prList.Items[0].HasStarted() && prList.Items[0].IsDone() && prList.Items[0].Status.GetCondition(apis.ConditionSucceeded).IsTrue() &&
+					prList.Items[1].HasStarted() && prList.Items[1].IsDone() && prList.Items[1].Status.GetCondition(apis.ConditionSucceeded).IsTrue()
```

Comment:
Unfortunately we cannot 100 % rely on that - in rare cases it could be also 3 snapshots (that's what the integration team told me). So I think it's safer if we verify which is the right one.